### DISCUSSION TITLE
两个小问题

### DISCUSSION THREAD

**Discussion by rockcitystore** [yihong0618/running_page#236] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/11556355/172557280-3b47e3c5-dc53-43b8-8e0a-e87a28626b45.png)
不太明白这里为什么取大于最近一次运动时间的记录 因为我是同步历史数据 所以把这里改成<lasttime了
另外就是历史记录大概700条 运行nike_to_strava_sync.py时限流10条 不太会python 不知道如何分批上传 一会研究看看 想问下这个10条的限制是strava要求的吗？

**Comment by yihong0618** [comment, marked as answer]
不是，之前的一个 bug 有时候会找了所有的 id, 用的 trick. 这块欢迎帮忙修复

**Comment by yihong0618** [comment]
2 会限流，starve 的 api 有限制。

**Reply by rockcitystore** [reply]
谢谢 我加个sleep试试 果然像geekplux说的一样 秒回哈哈哈哈

**Reply by yihong0618** [reply]
😂
